.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = clang
 	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
 	branch = clang_tot_upgrade
+[submodule "compiler-rt"]
```

Comment:
Iff the repo command allows an easy way to clone these submodules with depth=1, then yes.  For CI, all we care about for each submodule is the HEAD commit.  I feel like the [series of git clone commands](https://github.com/scchan/hcc/blob/add_compiler-rt/Jenkinsfile#L32) I hardcoded basically does what repo does.

I have tried (and I know Paul has tried to) to clone hcc with the --shallow-submodules flag, and it doesn't work.  We also tried a lot of different permutations with `git submodule update --depth=1`, but those also fail.